### DISCUSSION TITLE
清平调·一枝红艳露凝香@李白·唐

### DISCUSSION THREAD

**Discussion by aspoem-giscus** [meetqy/aspoem#7] [Announcements]
# 清平调·一枝红艳露凝香@李白·唐

一枝红艳露凝香，云雨巫山枉断肠。
借问汉宫谁得似，可怜飞燕倚新妆。 

https://aspoem.com/poem/6

**Comment by jackeyGao** [comment]
非常漂亮的网站！

**Reply by meetqy** [reply]
呀，原来是 chinese-poetry 作者，我就说咋这么眼熟。
